doc/maintaining/datastore.rst

```diff
@@ -232,6 +232,8 @@ alongside CKAN.
 
 To install this please look at the docs here: http://docs.ckan.org/projects/datapusher
 
+.. note:: The DataPusher only imports the first worksheet of a spreadsheet. It does also
```

Comment:
You might consider moving the "also":

"It also does not support duplicate column headers."

Reply:
We could additionally clarify that blank column headings implies duplicate column headings, e.g.

> It also does not support duplicate column headers. That includes blank column headings.